extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -80,6 +80,11 @@ export class VisibilityModel {
               `Cannot repeat with interval less than ${MIN_REPEAT_INTERVAL}, ` +
               ' repeat set to false');
         }
+      } else {
```

Comment:
FYI going to kill the whole if/else on line 65 next, and make repeat only be allowed to be a boolean, not a number. But probably won't get to that before the canary that's in 2 hours.

Reply:
you might want to keep this PR simple for easy cherry-pick

Reply:
Done